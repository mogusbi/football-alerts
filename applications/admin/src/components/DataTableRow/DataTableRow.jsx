import TableRow from '@material-ui/core/TableRow';
import React, {memo} from 'react';

function DataTableRow (props) {
  return (
    <TableRow {...props} />
  );
}

export default memo(DataTableRow);
