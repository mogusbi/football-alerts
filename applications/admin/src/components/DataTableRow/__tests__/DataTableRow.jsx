import {mount} from 'enzyme';
import React from 'react';
import DataTableRow from '../DataTableRow';

describe('DataTableRow', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(
      <table>
        <tbody>
          <DataTableRow />
        </tbody>
      </table>
    );
  });

  it('should render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
