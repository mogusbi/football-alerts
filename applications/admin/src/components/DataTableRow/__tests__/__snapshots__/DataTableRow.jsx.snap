// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTableRow should render component 1`] = `
<table>
  <tbody>
    <DataTableRow>
      <WithStyles(TableRow)>
        <TableRow
          classes={
            Object {
              "footer": "MuiTableRow-footer-5",
              "head": "MuiTableRow-head-4",
              "hover": "MuiTableRow-hover-3",
              "root": "MuiTableRow-root-1",
              "selected": "MuiTableRow-selected-2",
            }
          }
          component="tr"
          hover={false}
          selected={false}
        >
          <tr
            className="MuiTableRow-root-1"
          />
        </TableRow>
      </WithStyles(TableRow)>
    </DataTableRow>
  </tbody>
</table>
`;
